addons/better-featured-project/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Profile page banner",
-  "description": "Makes the featured project on any user's profile page into a nice-looking banner.",
+  "description": "Makes the featured project on any user's profile into a nice-looking banner.",
```

Comment:
Now that we're on it, can we use a better verb? "Makes" is kinda weird here. "Converts" looks more suitable, maybe "changes", not sure.

Reply:
`Turns`?

Reply:
I like "turns". "Makes" would also be fine if "into" was removed.